src/config.ts

```diff
@@ -1,30 +1,63 @@
 import { Args } from "grimoire-kolmafia";
-import { Item } from "kolmafia";
+import { Item, print } from "kolmafia";
 import { $item, $items, get } from "libram";
 
 const workshedAliases = [
-  { item: $item`cold medicine cabinet`, aliases: ["cmc"] },
-  { item: $item`model train set`, aliases: ["trainset", "trainrealm", "train"] },
-  { item: $item`Asdon Martin keyfob`, aliases: ["breadcar", "car", "asdon", "aston"] },
-  { item: $item`diabolic pizza cube`, aliases: ["cube", "pizza", "pizzacube"] },
-  { item: $item`portable Mayo Clinic`, aliases: ["mayo", "clinic"] },
-  { item: $item`Little Geneticist DNA-Splicing Lab`, aliases: ["dna", "dnalab"] },
+  { item: $item`model train set`, aliases: ["trainrealm"] },
+  { item: $item`Asdon Martin keyfob`, aliases: ["breadcar", "car", "aston"] },
+  { item: $item`Little Geneticist DNA-Splicing Lab`, aliases: ["dnalab"] },
 ];
-const unaliasedSheds = $items`spinning wheel, warbear auto-anvil, warbear chemistry lab, warbear high-efficiency still, warbear induction oven, warbear jackhammer drill press, warbear LP-ROM burner`;
+const unaliasedSheds = $items`cold medicine cabinet, diabolic pizza cube, portable Mayo Clinic, spinning wheel, warbear auto-anvil, warbear chemistry lab, warbear high-efficiency still, warbear induction oven, warbear jackhammer drill press, warbear LP-ROM burner`;
+const allWorkshedAliases = [
+  ...workshedAliases.map(({ item, aliases }) => {
+    return { item: item, aliases: [...aliases, item.name.toLowerCase()] };
+  }),
+  ...unaliasedSheds.map((item) => {
+    return { item: item, aliases: [item.name.toLowerCase()] };
+  }),
+];
+
+function toInitials(s: string): string {
+  const initials = s
+    .split(" ")
+    .map((term) => term[0])
+    .join("");
+  return initials.length >= 3 ? initials : "";
+}
+
+function stripString(s: string): string {
+  if (s.includes(" ")) return stripString(s.replace(" ", ""));
+  else if (s.includes("-")) return stripString(s.replace("-", ""));
+  return s;
+}
```

Comment:
Instead of this bespoke function, use either replaceAll (may need a polyfill?) or a regexp with the `g` flag in `replace`

Reply:
I will probably do something about this Some Day